config/crd/bases/workload.nephio.org_amfdeployments.yaml

```diff
@@ -0,0 +1,366 @@
+                  type: object
+                  x-kubernetes-map-type: atomic
+                type: array
+              interfaces:
+                description: Interfaces defines the interfaces associated with the
+                  NF deployment
+                items:
+                  description: InterfaceConfig defines the configuration of the interface
+                  properties:
+                    ipv4:
+                      description: IPv4Config defines the ipv4 configuration of the
+                        interface
+                      properties:
+                        address:
+                          description: Address defines the IPv4 address and prefix
+                            length in CIDR notation [IP prefix, range IPv4 with host
+                            bits]
+                          type: string
+                        gateway:
+                          description: Gateway defines the IPv4 address associated
+                            to the interface as a gateway
+                          type: string
+                      required:
+                      - address
+                      type: object
+                    ipv6:
+                      description: IPv6Config defines the ipv6 configuration of the
+                        interface
+                      properties:
+                        address:
+                          description: Address defines the IPv6 address and prefix
+                            length in CIDR notation [IP prefix, range IPv6 with host
+                            bits]
+                          type: string
+                        gateway:
+                          description: Gateway defines the IPv6 address associated
+                            to the interface as a gateway
+                          type: string
+                      required:
+                      - address
+                      type: object
+                    name:
+                      description: Name defines the name of the interface
+                      maxLength: 253
+                      minLength: 1
+                      type: string
+                    vlanID:
+                      description: VlanID defines the specific vlan id associated
+                        on this interface
+                      type: integer
+                  required:
+                  - name
+                  type: object
+                type: array
+              networkInstances:
+                description: NetworkInstances defines the network instances associated
+                  with the NF deployment
+                items:
+                  description: A networkInstance is a Layer 3 forwarding construct
+                    such as a virtual routing and forwarding (VRF) instance,
+                  properties:
+                    bgp:
+                      description: bgp defines the BGP configuration associated with
```

Comment:
is the bgp session part of the CNF or in the worker node ?..since we have made this generic, is BGP config relevant to all the NF of our interest i.e  AMF, SMF, UPF..?